app/assets/javascripts/components/reward_page_components/EditPersonLink.jsx

```diff
@@ -0,0 +1,33 @@
+class EditPersonLink extends React.Component {
```

Comment:
Waarschijnlijk zie ik iets over het hoofd, maar volgens mij wordt dit React component nergens gebruikt?

Reply:
nee klopt, maar ik denk dat we hem straks wel willen toevoegen aan de rewards pagina. Het lijkt me eigenlijk de default dat ze daar een edit profile link moeten kunnen hebben. Ik heb hem voor nu niet toegevoegd omdat ze nu expliciet vroegen om het niet te hebben.